test/runtime_test.cc

```diff
@@ -245,6 +247,40 @@ TEST_P(TestVM, Callback) {
   ASSERT_EQ(res.u32(), 100100); // 10000 (global) + 100(in callback)
 }
 
+TEST_P(TestVM, TerminateExecution) {
+  if (engine_ != "v8") {
+    return;
```

Comment:
Add note or a `TODO` explaining why this is executed only with V8 and when/if the same functionality is going to be added to other engines.

Reply:
TODO added.